shoov/modules/custom/shoov_repository/shoov_repository.module

```diff
@@ -167,3 +167,47 @@ function shoov_repository_check_repository_unique($github_id, $node) {
 
   throw new Exception(format_string('GitHub ID @githubid was duplicated.', array('@githubid' => $github_id)));
 }
+
+/**
+ * Check .shoov.yml file exists in repository.
+ *
+ * @param $node
+ *  Repository node
+ * @param $branch
+ *  The Git branch name.
+ *
+ * @return bool
+ *  TRUE if file exists.
+ */
+function shoov_repository_config_file_exists($node, $branch) {
+  if ($node->type != 'repository') {
+    return FALSE;
+  }
+
+  if (variable_get('test_mode', FALSE)){
+    // This is the test mode. Don't check the repo.
+    return TRUE;
+  }
+
+  global $user;
```

Comment:
Because could be that the repository node was already created by other user earlier and we don't want to use someone else's access token.